### ISSUE TITLE
create accessibility details file

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#844]
Following up on #794 - The complete details of the detected violations should be then dumped into a single large file in the same s3 bucket as the primary and all snapshots, and called perhaps `accessbiility_details.json`

The database field maybe makes sense to be accessibility_violations_details or the like.   Or the results could just be captured in a temporary stream each scan day.  Whatever is most native.
